src/main.cpp

```diff
@@ -431,7 +432,7 @@ static void parse_arguments(int argc, char** argv, Globals& g) {
     // parse options
     while (true) {
         int option_index = 0;
-        int c = getopt_long(argc, argv, "+c:vl", long_options, &option_index);
+        int c = getopt_long(argc, argv, "+c:vh", long_options, &option_index);
```

Comment:
I'm not totally sure about this syntax, but shouldn't the `l` have stayed here?

Reply:
Oh right. I've accidentally deleted it...